druid/src/main/java/org/apache/calcite/adapter/druid/DruidQuery.java

```diff
@@ -980,35 +997,43 @@ private JsonFilter translateFilter(RexNode e) {
         final boolean numeric =
             call.getOperands().get(posRef).getType().getFamily()
                 == SqlTypeFamily.NUMERIC;
+
+        final ExtractionFunction extractionFunction = ExtractionFunctionUtil.buildExtraction(call
+            .getOperands().get(posRef));
+        String dimName = tr(e, posRef);
+        if (dimName.equals(DruidConnectionImpl.DEFAULT_RESPONSE_TIMESTAMP_COLUMN)) {
+          dimName = "__time";
```

Comment:
Please, use DruidTable.DEFAULT_TIMESTAMP_COLUMN here and leave a comment explaining why we are renaming the dimension column.